documentation/milestones/individual_milestones/communication_platforms.md

```diff
@@ -141,65 +143,72 @@ Ideally, the tool would have the ability to designate some users as direct edito
   - [ ] Instructions for the internal team that assists with onboarding new users are clearly and accessibly documented in a public place.
   - [ ] Code for managing and deploying these services is deployed to `main` & PROD (if necessary)
   - [ ] An ADR has been recorded which documents the tool chosen and the reasons for selecting it
+  - [ ] An ADR has been recorded which outlines the norms around how these tools should be used collectively as part of a broader team communication plan
+- [ ] Ticketing system is live with the following conditions satisfied:
+  - [ ] At least 1 ticket has been created and assigned to an epic (or an equivalent concept).
+  - [ ] Without logging in, members of the public can see tickets that are being worked on.
+  - [ ] Default 'templates' have been created for tickets, epics, etc, that have the fields we've decided are necessary for the project.
+  - [ ] Users can track tickets in sprints and monitor things like burndown.
+  - [ ] Members of the public can submit feature requests or report bugs
```

Comment:
are story points and burndown charts necessary conditions, or more of a nice to have?